docs/GUIDE.md

```diff
@@ -688,7 +688,7 @@ just x = \f _ -> f x
 nothing :: Maybe a
 nothing = \_ n -> n
 
-foo :: Maybe a ->
+foo :: Maybe a -> a
```

Comment:
ah, my bad. This should `Maybe Integer -> Integer` - could you change that please?

Reply:
done